test/unit/views/main-view.js

```diff
@@ -250,12 +251,20 @@ describe('MainView', function () {
       PayPalView
     ].forEach(function (View) {
       describe('when given a ' + View.ID + 'view', function () {
+        beforeEach(function () {
+          this.clock = sinon.useFakeTimers();
```

Comment:
If you use `this.sandbox`, it should be restored automatically in the global `afterEach`.